src/components/molecules/Visualizer/hooks.ts

```diff
@@ -257,6 +260,12 @@ export default ({
     alignSystem,
   });
 
+  const { pluginInstances } = usePluginInstances({
+    alignSystem,
+    floatingWidgets,
+    blocks: infobox?.blocks ?? selectedLayer?.infobox?.blocks,
```

Comment:
Doesn't `infobox` get assigned from `selectedLayer?.infobox` in `useLayers`? If that is correct, we don't need the fallback because they are the same thing

Reply:
Yes you're right. Should not use `infobox?.blocks` here. Updated.